luatest/assertions.lua

```diff
@@ -568,6 +575,7 @@ end
 -- @param value
 -- @string[opt] message
 function M.assert_eval_to_true(value, message, ...)
+    log.info('Assert eval to true %s', value)
```

Comment:
I propose to use the following format:

`` Assert `%s` ``

Reply:
Check out my comment